src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift

```diff
@@ -14,7 +14,7 @@ class DataDonationViewController: DynamicTableViewController, DeltaOnboardingVie
 		largeTitleDisplayMode: UINavigationItem.LargeTitleDisplayMode = .never
 	) {
 		self.viewModel = viewModel
-		self.largeTitleDisplayMode = largeTitleDisplayMode
+		self.largeTitleDisplayMode = .never
```

Comment:
The parameter is still there, so it seems confusing to me that it is not used

Reply:
The DataDonationViewController is initialized also in the on boarding, im confirming with Kerstin if it should change as well

Reply:
I remove the variable and also added an extra space to the legal text in the on boarding so we match the figma design